server/controllers/logoutController.js

```diff
@@ -0,0 +1,4 @@
+exports.logoutRouter = (req, res) => {
+  res.clearCookie('authToken');
+  res.redirect('/');
```

Comment:
You shouldn't respond with a redirect from the backend, you have to send a response with a prober message in order to handle it correctly at the backend
```suggestion
  res.clearCookie('authToken').json({ statusCode: 200, message: 'Logged out successfully' });
```